tools/reviewer/aa/commands/DiffCommand.java

```diff
@@ -178,5 +184,38 @@ public boolean run(String[] args) {
     codeReviewBlockingStub.createDiff(request);
     return true;
   }
+
+  private void addGithubRepo(Diff.Builder diffBuilder) {
+    List<String> alreadySettingGithubRepoNames =
+        diffBuilder
+            .getGithubPrList()
+            .stream()
+            .map(githubPr -> githubPr.getRepo().getRepo())
+            .collect(Collectors.toList());
+    try {
+      fileUtils
+          .listContents(workspacePath)
+          .stream()
+          .map(path -> fileUtils.joinToAbsolutePath(workspacePath, path))
+          .filter(fileUtils::folderExists)
+          .forEach(
+              path -> {
+                String repoName = Paths.get(path).getFileName().toString();
+                GitRepo repo = this.gitRepoFactory.create(path);
+                String repoURL = repo.getRemoteURL();
```

Comment:
I think you should get the `repoName` from the URL, like you do with `repoOwner`.
You could add a warning here if they're not the same.